src/main/kotlin/io/polygon/kotlin/sdk/rest/Trades.kt

```diff
@@ -56,7 +56,7 @@ data class TradesResponse(
     val status: String? = null,
     @SerialName("request_id") val requestID: String? = null,
     @SerialName("next_url") override val nextUrl: String? = null,
-    override val results: List<Trade> = emptyList()
+    override val results: List<Trade>? = emptyList()
```

Comment:
This is technically a breaking change, is there a particular reason to change this or can we leave it how it is? 

```suggestion
    override val results: List<Trade> = emptyList()
```

Reply:
No particular reason, will revert since its breaking.